internal/pkg/cli/pipeline_init.go

```diff
@@ -318,6 +400,35 @@ func (o *initPipelineOpts) askPipelineName() error {
 	return nil
 }
 
+func (o *initPipelineOpts) askOrValidatePipelineType() error {
+	if o.pipelineType != "" {
+		for _, typ := range pipelineTypes {
+			if o.pipelineType == typ {
+				return nil
+			}
+		}
+		return fmt.Errorf("invalid pipeline type %q; must be one of %s", o.pipelineType, english.WordSeries(quoteStringSlice(pipelineTypes), "or"))
+	}
+
+	typ, err := o.prompt.SelectOption("What type of continuous delivery pipeline is this?",
```

Comment:
Adding a new prompt means that saved pipeline init commands which executed previously will now require an additional user input, which means we may have to mark this release as containing a breaking change.

Just want to make sure we know to highlight it in the release docs if that's the case.

Reply:
👍🏼 ! Fortunately (typically) `copilot pipeline init` should not be scripted and hopefully nobody does that :crylaugh: